src/net.cpp

```diff
@@ -911,7 +911,7 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     if (NetPermissions::HasFlag(permissionFlags, NetPermissionFlags::PF_ISIMPLICIT)) {
         NetPermissions::ClearFlag(permissionFlags, PF_ISIMPLICIT);
         if (gArgs.GetBoolArg("-whitelistforcerelay", false)) NetPermissions::AddFlag(permissionFlags, PF_FORCERELAY);
-        if (gArgs.GetBoolArg("-whitelistrelay", false)) NetPermissions::AddFlag(permissionFlags, PF_RELAY);
+        if (gArgs.GetBoolArg("-whitelistrelay", true)) NetPermissions::AddFlag(permissionFlags, PF_RELAY);
```

Comment:
Add comment `DEFAULT_WHITELISTRELAY = true`, so if anyone changes that default they won't miss this spot.

Reply:
Actually I can easily move the constant (https://github.com/bitcoin/bitcoin/pull/16631#discussion_r314708245) testing it.